nightingale_payload/package.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>nightingale_payload</name>
```

Comment:
nit: Could there be a better name for this package? I feel like payload isn't specific enough to mean the box. For communications "payload" is also used a lot for data storage and transfer. Could we maybe do like "payload_sensing"? Not the most important thing though since we all know what it means

Reply:
ill remove this pkg from this pr since its not v0